main/cmds.go

```diff
@@ -92,22 +94,37 @@ func migrateToMostRecentSequence(ctx *log.Context, h HandlerEnvironment, seqNum
 	// do not have invent another method.  The CustomScript extension should have been using this
 	// from the beginning, but it was not.
 	//
-	computedSeqNum, err := FindSeqNumStatus(h.HandlerEnvironment.StatusFolder)
+	ctx.Log("event", "migrate to mrseq", "message", fmt.Sprintf("cannot find mrseq at %v", mostRecentSequence))
+	fout, err := os.Create(mostRecentSequence)
+	defer fout.Close()
 	if err != nil {
-		// If there was an error, the sequence number is zero.
 		ctx.Log("event", "migrate to mrseq", "error", err)
-		return
 	}
 
-	fout, err := os.Create(mostRecentSequence)
-	if err != nil {
-		ctx.Log("event", "migrate to mrseq", "error", err)
-		return
+	if seqNum != 0 {
+		computedSeqNum := seqNum
+		_, err := fout.WriteString(fmt.Sprintf("%v", computedSeqNum))
+		if err == nil {
+			ctx.Log("event", "migrated to mrseq", "message", fmt.Sprintf("migrated mrseq to %v from config seqnum", computedSeqNum))
+			return 
+		} 
+		//if error writing seqNum, then try getting seq num from status 
```

Comment:
so, if the above generates an error, that means we couldn't write the new mrseq, right? In that case, I don't think it will help to attempt to recompute it, because that's not where the problem was. Clearly something was holding a lock to prevent us from writing, so we should probably write out the error in that case and presumably fail the install since we can no longer guarantee the seqNo.

Reply:
That makes sense. In that case, I think that seqNo should always have some value to it based on the logic in main.go, so would checking status folders still be necessary?

Reply:
func migrateToMostRecentSequence should also return an error if it fails to write to mrseq file and fail the installation.